fab/operations/release.py

```diff
@@ -143,10 +144,14 @@ def offline_pip_install():
         env.sudo_user, env.virtualenv_root
     )
     requirements = os.path.join(env.code_root, 'requirements')
+    requirements_file = os.path.join(requirements, 'requirements.txt')
+    for offline_lib in ['pygooglechart', 'django-transfer', 'django-two-factor-auth', 'pyzxcvbn']:
```

Comment:
👍 frustrating, but seems like a good work around. gonna be fun when someone upgrades one of these...